### PR TITLE
Add support for cli network prune --filter flag

### PR DESCRIPTION
**Pull request by jmguzik**
Missing cli functionality added.
/kind feature

### CODE REVIEW

File: docs/source/markdown/podman-network-prune.1.md [hunk 1]
```diff
@@ -16,6 +16,20 @@ the so-called default network which goes by the name of *podman*.
 
 Do not prompt for confirmation
 
+#### **\-\-filter**
+
+Filter output based on conditions given.
+Multiple filters can be given with multiple uses of the --filter flag.
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-network-prune.1.md:22]:
```suggestion
Multiple filters can be given with multiple uses of the --filter option.
```

**Comment by TomSweeneyRedHat** [comment]
All kinds of unhappy red tests @jmguzik

File: docs/source/markdown/podman-network-prune.1.md (hunk 1 above)
Comment (jmguzik) [reply docs/source/markdown/podman-network-prune.1.md:22]: Ok. Taken almost entirely from another place (network list). Correct there as well?

**Comment by jmguzik** [comment]
> All kinds of unhappy red tests @jmguzik

Does not work as expected... I will try to have a closer look in the next few days.

**Comment by rhatdan** [comment]
LGTM
/approve

**Pull request merged**